src/Resources/views/Form/form_admin_fields.html.twig

```diff
@@ -357,7 +357,7 @@ file that was distributed with this source code.
 
 {% block form_row %}
     {% set show_label = show_label ?? true %}
-    <div class="form-group{% if errors|length > 0 %} has-error{% endif %}" id="sonata-ba-field-container-{{ id }}">
+    <div{% with {attr: row_attr|merge({id: 'sonata-ba-field-container-'~id, class: (row_attr.class|default('') ~ ' form-group' ~ ((not compound or force_error|default(false)) and not valid ? ' has-error'))|trim})} %}{{ block('attributes') }}{% endwith %}>
```

Comment:
This line is hard to read. Please wrap some parts

Reply:
I split this line in some variables